dev/CommandBarFlyout/CommandBarFlyoutCommandBar.cpp

```diff
@@ -489,45 +493,35 @@ void CommandBarFlyoutCommandBar::AddShadow()
 {
     if (SharedHelpers::IsThemeShadowAvailable())
     {
-#ifdef USE_INSIDER_SDK
         //Apply Shadow on the Grid named "ContentRoot", this is the first element below
         //the clip animation of the commandBar. This guarantees that shadow respects the 
         //animation
         winrt::IControlProtected thisAsControlProtected = *this;
         auto grid = GetTemplateChildT<winrt::Grid>(L"ContentRoot", thisAsControlProtected);
-        if (winrt::IUIElement10 grid_uiElement10 = grid)
```

Comment:
Hmm, I think we actually want this code. See PR #281

I manually cherry picked the CommandBarFlyout changes and made some update in the same PR, that's probably why it's messed up. Is it possible to redo the port and "keep current"?

Reply:
Sure, I can update this.